exporter/metric/metric.go

```diff
@@ -337,9 +338,19 @@ func (attrs *attributes) GetString(key string) (string, bool) {
 //
 // https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.monitoredResourceDescriptors
 func (me *metricExporter) resourceToMonitoredResourcepb(res *resource.Resource) *monitoredrespb.MonitoredResource {
-	gmr := resourcemapping.ResourceAttributesToMonitoredResource(&attributes{
-		attrs: attribute.NewSet(res.Attributes()...),
-	})
+	attrs := &attributes{attrs: attribute.NewSet(res.Attributes()...)}
+	cloudPlatform, _ := attrs.GetString(string(semconv.CloudPlatformKey))
+	var gmr *resourcemapping.GceResource
+	switch cloudPlatform {
+	case semconv.CloudPlatformGCPCloudRun.Value.AsString():
+		fallthrough
+	case semconv.CloudPlatformGCPCloudFunctions.Value.AsString():
```

Comment:
is there a tradeoff between `fallthrough` here and just making it a single case with multiple matches?

Reply:
Ah, thats a good point.

Reply:
done